docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-112.adoc

```diff
@@ -0,0 +1,49 @@
+== Azure Functions HTTP Trigger Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_112
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An Azure Functions HTTP Trigger Key is essential for securing HTTP-triggered Azure Functions, allowing only authorized clients to invoke the function. This key ensures controlled access, which is crucial for preventing unauthorized data exposure and manipulation through the function. Secure management of this key is imperative to maintain the integrity and security of the applications reliant on Azure Functions.
+
+=== Fix - Buildtime
+
+*Azure*
+
+If an Azure Functions HTTP Trigger Key is inadvertently exposed, immediate actions should be taken to revoke the old key and generate a new one, thus securing access to the function:
+
+1. Log into your Azure Portal using your administrator credentials.
+
+2. Navigate to the Azure Functions app in question.
+
+3. Access the 'Function keys' section within the platform features of the function.
+
+4. Delete the exposed key and generate a new one by selecting '+ New key'. Give the new key a name and save the changes.
```

Comment:
Delete the exposed key and generate a new one by selecting '+ New key'. Provide a name for the new key and save the changes.